pkg/cli/updater/updater.go

```diff
@@ -33,10 +33,19 @@ import (
 	"golang.org/x/term"
 )
 
-// Disabled globally disables the automatic updater. This is helpful
-// for when using an external package manager, such as brew.
-// This should usually be done with an ldflag.
-var Disabled bool
+// Disabled globally disables the automatic updater.
+// This is helpful for when using an external package manager, such as brew.
+// This should usually be done with an ldflag:
+//
+//	go run -ldflags "-X github.com/getoutreach/gobox/pkg/cli/updater.Disabled=true" ...
+//
+// or you can do it in main() before UseUpdater is called:
+//
+//	updater.Disabled="true"
+//
+// Any value other than "true" is considered false.
+// The type must be string to allow -ldflags "-X ..."
```

Comment:
Nit: code style, grammar
```suggestion
//	updater.Disabled = "true"
//
// Any value other than "true" is considered false.
// The type is string to allow for invoking via `go run -ldflags "-X ..."`.
```